public/app/features/explore/TraceView/TraceView.tsx

```diff
@@ -245,7 +246,9 @@ export function TraceView(props: Props) {
           />
         </>
       ) : (
-        <div className={styles.noDataMsg}>No data</div>
+        <div className={styles.noDataMsg}>
```

Comment:
Are translations available to plugins? There were some ideas to externalize some visualizations. cc @adrapereira

Reply:
we're currently working on that functionality right now 👍 until then, anything that is externalised will need the translations stripping out - there will be a failing `Trans`/`t` import anyway